extra/stoatLordCardStuff.js

```diff
@@ -31,6 +31,33 @@ async function load() {
         cardFormated.attack = card["Power"]
         cardFormated.health = card["Health"]
 
+        // parsing cost
+        card["Cost"] = card["Cost"]
```

Comment:
Missing a `.toLowerCase()`. Without lowering them most of the case won;t be match. Aug doesn't require this because it is already lower, though I should add it

Reply:
I see can fix that rq